sys/kern/pool.c

```diff
@@ -242,16 +247,20 @@ static void pool_dtor(pool_t *pool) {
   klog("destroyed pool '%s' at %p", pool->pp_desc, pool);
 }
 
-static void pool_init(pool_t *pool, const char *desc, size_t size) {
+static void pool_init(pool_t *pool, const char *desc, size_t size,
```

Comment:
Consider adding a comment explaining each of the parameters.
I mean I'm not sure if it's needed, but there are 4 of them which is a quite

Reply:
This is a static function, hence is used only within the file which defines it.